python/cac_tripplanner/templates/place-detail.html

```diff
@@ -1,6 +1,30 @@
 
 {% extends "base.html" %}
 {% load staticfiles %}
+
+{% block pagetitle %}
+<title>{% block title %}GoPhillyGo | {{ destination.name }}{% endblock %}</title>
+
+<meta property="og:title" content="{{ destination.name }}" />
+<meta name="twitter:title" content="{{ destination.name }}" />
+{% endblock %}
+
+{% block extrametatags %}
+<meta property="fb:app_id" content="{{ fb_app_id }}" />
+<meta property="og:url"
+    content="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}" />
```

Comment:
Not directly, as far as I am aware. `is_secure` seems to be [the recommended](http://stackoverflow.com/questions/1841826/can-you-check-the-internet-protocol-from-djangos-template) thing to use.